maintainers/maintainer-list.nix

```diff
@@ -5714,6 +5714,12 @@
     githubId = 115218;
     name = "Felix Richter";
   };
+  malvo = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add malvo``.